### PR TITLE
feat(clerk-js): Introduce InputGroup internal form component

### PR DESCRIPTION
**Pull request by panteliselef**
## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `@clerk/chrome-extension`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

image

### CODE REVIEW

File: packages/clerk-js/src/ui/elements/InputGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    borderTopRightRadius: '0',
+    borderBottomRightRadius: '0',
+    width: 'fit-content',
+    display: 'flex',
+    alignItems: 'center',
+  });
+
+  return (
+    <Flex
+      // TODO: update this.
```
Comment (anagstef) [inline packages/clerk-js/src/ui/elements/InputGroup.tsx:39]: 🔧  Let's add a more descriptive message on this `TODO`

File: .changeset/tall-socks-teach.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (anagstef) [inline .changeset/tall-socks-teach.md:1]: ❓ Will these changes impact something that is already in production?

**Review by anagstef — APPROVED** [review]

File: .changeset/tall-socks-teach.md (hunk 2 above)
Comment (panteliselef) [reply .changeset/tall-socks-teach.md:1]: Nothing was released

**Comment by clerk-cookie** [comment]
This PR has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request merged**